pkg/private/pkg_files.bzl

```diff
@@ -133,11 +134,14 @@ def _process_pkg_filegroup(content_map, pkg_filegroup_info, origin, default_mode
     for psl in pkg_filegroup_info.pkg_symlinks:
         _process_pkg_symlink(content_map, psl[0], psl[1], default_mode, default_user, default_group)
 
-def process_src(content_map, src, origin, default_mode, default_user, default_group):
+def process_src(ctx, content_map, files, src, origin, default_mode,
+                default_user, default_group):
     """Add an entry to the content map.
 
     Args:
-      content_map: The content map
+      ctx: rule context
```

Comment:
`ctx` appears unused.  Consider removing it, or documenting this fact.

Reply:
That was left over from another idea.  Removed.